lnwallet/btcwallet/btcwallet.go

```diff
@@ -656,14 +656,17 @@ func (b *BtcWallet) SubscribeTransactions() (lnwallet.TransactionSubscription, e
 // it has fully synced to the current best block in the main chain.
 //
 // This is a part of the WalletController interface.
-func (b *BtcWallet) IsSynced() (bool, error) {
+func (b *BtcWallet) IsSynced() (bool, int64, error) {
+	var err error
 	// Grab the best chain state the wallet is currently aware of.
+	// also grab the timestamp of the best block to return for determining
+	// sync progress
 	syncState := b.wallet.Manager.SyncedTo()
+	walletBestBlock, err := b.chain.GetBlockHeader(&syncState.Hash)
```

Comment:
The error isn't being checked here. Also you don't need to declare the error at the top of the scope.